tests/kas_default_scenario.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { test, expect } from '@playwright/test';
```

Comment:
I guess this file should be moved under rhosak folder

Reply:
moved

Reply:
Can I resolve this conversation @kornys ?